.github/workflows/deployment-development.yml

```
@@ -0,0 +1,54 @@
+# Deployment workflow to be used for Jira reporting
+name: Deploy alpha to Development
+
+on:
+  push:
+    branches: ["alpha"]
+
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  deploy-dev:
+    name: Deploy to Dev environment
+    # IMPORTANT: the workflow must have write access to deployments, otherwise the action will fail.
+    permissions:
+      deployments: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Please change this to use the latest version
```suggestion
      - uses: actions/checkout@v3
```